src/service.js

```diff
@@ -166,10 +131,16 @@ export function installServiceInEmbedScope(embedWin, id, service) {
       'Service override can only be installed in embed window: %s', id);
   dev().assert(!getLocalExistingServiceForEmbedWinOrNull(embedWin, id),
       'Service override has already been installed: %s', id);
-  getServiceInternal(embedWin, embedWin, id, undefined, () => service);
+  registerServiceInternal(
+      embedWin,
+      embedWin,
+      id,
+      /* opt_ctor */ undefined,
+      () => service);
+  // Force service to build
+  getServiceInternal(embedWin, embedWin, id);
```

Comment:
This can be reverted. Let's address the future of the convenience builder separately.